scripts/compose.py

```diff
@@ -930,7 +930,7 @@ def _content(self):
             logging=None,
             environment={
                 "ELASTIC_APM_SERVICE_NAME": "rum",
-                "ELASTIC_APM_SERVER_URL": self.options.get("apm_server_url", DEFAULT_APM_SERVER_URL),
+                "APM_SERVER_URL": self.options.get("apm_server_url", DEFAULT_APM_SERVER_URL),
```

Comment:
I think `ELASTIC_APM_SERVER_URL` is a standard among other images!

Reply:
We don't even use it anywhere..

Reply:
on the opbeans projects we use ELASTIC_APM_SERVER_URL

https://github.com/elastic/opbeans-node

Reply:
Removed this change